submissions/cynthiaNwakaeme/php-database/cookies.php

```diff
@@ -3,53 +3,58 @@
 
 echo "<h2> List of Customers at Taries Empire </h2>";
 
-try{
-    /* ACCESSING E-COMMERCE DATABASE AND QUERYING THE DATABASE*/
-    $use_db = "USE taries_empire";
-    $sql = "SELECT * FROM customers";
-    $pdo->exec($use_db);
-    $stmt = $pdo->query($sql);
-    
-
-    /* GENERATING COOKIES
-    1. result is initalized to get all records from database
-    2. This result is serialize to change the array/object to simple string/text file which is required to create cookies.
-     */
-     $result = $stmt->fetchAll();
-    setcookie("customer", serialize($result), time() + (86400 * 30), "/", "", false); //60sec *60mins *24hours/day => 86400 = 1 day
-
-    // To display results on the browser as a table, we convert the result to array/object.
-    $output = unserialize($_COOKIE["customer"]);
+        try{
+            
+            /* ACCESSING E-COMMERCE DATABASE AND QUERYING THE DATABASE*/
+            $use_db = "USE taries_empire";
+            $sql = "SELECT * FROM customers";
+            $pdo->exec($use_db);
+            $stmt = $pdo->query($sql);
+            
+        
+            /* GENERATING COOKIES
+            1. result is initalized to get all records from database
+            2. This result is serialize to change the array/object to simple string/text file which is required to create cookies.
+             */
+             $result = $stmt->fetchAll();

```

Comment:
from line 9 - 19 will always get executed on a page reload

Reply:
I will advise you put your query on a separate function, and always call that function if the cookie is not set

Reply:
Thank you.
I will do that. But i want to close this PR please.